package.json

```diff
@@ -51,7 +51,8 @@
     "react-markdown": "^7.0.0",
     "react-portal": "^4.2.1",
     "superstruct": "^0.15.3",
-    "ts-node": "^10.2.1",
+    "ts-node": "^10.4.0",
+    "tsconfig-paths": "^3.12.0",
```

Comment:
These should probably be in `devDependencies` but I didn't want to break stuff on heroku if it's running in `NODE_ENV=production`

Reply:
We use Vercel so it should be ok